### ISSUE TITLE
Interfacing with the real ethereum blockchain

### ISSUE THREAD

**Issue by o-jasper** [kumavis/ethereum-simulator#21]
You could have a 'simulation mode' where you take out the block mechanism and replace it with a 'fake compliant' one.

Ethereum scripts, fees would automatically work the same as in ethereum because it essentially _is_ ethereum.

**Comment by k1n0k0** [comment]
Yeah this idea has come up a few times, I think mostly b/c the current cpp client's UI is ugly and unperformant. The core team has a standalone webkit-based ui in the works, to be built on top of the cpp client. I will continue to consider this idea, but I am planning to develop this more in the direction of an IDE. That said, with contracts able to call contracts, and the recent protocol changes to encourage this behaviour in the name of high-level coding via abstraction, being able to reference existing contracts from the IDE would be a significant value add.

**Comment by k1n0k0** [comment]
Renamed issue to: `Interfacing with the real ethereum blockchain`
Original title: `Running actual ethereum with this as interface?`
